include/pokeagb/pokemon/move.h

```diff
@@ -368,7 +368,325 @@ enum Move {
     MOVE_SHOCK_WAVE,
     MOVE_WATER_PULSE,
     MOVE_DOOM_DESIRE,
-    MOVE_PYSCHO_BOOST,
+    MOVE_PSYCHO_BOOST,
```

Comment:
Please remove - anything that's not in a base generation 3 ROM should not be added and should be overridden in your own code